client/components/product.js

```diff
@@ -1,16 +1,36 @@
 import React, {Component} from 'react';
 import {connect} from 'react-redux';
 import {withRouter} from 'react-router-dom';
+import axios from 'axios';
+import { postProd_OrderThunker } from '../store';
 
 export class productComponent extends Component {
 
   constructor(props) {
     super(props);
     this.quantities = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+    //this.handleClick = this.handleClick.bind(this);
   }
+
+  // handleClick(event) {
+  //   event.preventDefault();
+  //   // Is this^ line necessary?
+  //   if (this.props.cartId) {
+  //     //add to the current cart
+  //     // orderProduct put route
+  //     props.dispatch()
+  //     // axios.post('/api/order_product')
+  //     // .catch(err => console.log(err))
+  //   } else {
+  //     // make a cart and add it
+  //   }
+  //   console.log(event.target)
+  // }
```

Comment:
Remember that commented out code should not be in master, which it will be if I merge this PR as it is.